service/controller/appcatalog/resource/appcatalogentry/create.go

```diff
@@ -34,19 +34,6 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
-	// Skip creating appcatalogentry CRs if the catalog is not public.
-	if key.AppCatalogVisibility(cr) != publicVisibilityType {
-		r.logger.Debugf(ctx, "not creating CRs for catalog %#q with visibility %#q", cr.Name, key.AppCatalogVisibility(cr))
-		r.logger.Debugf(ctx, "canceling resource")
-		return nil
-	}
-	// Skip creating appcatalogentry CRs if this is a community catalog.
-	if key.AppCatalogType(cr) == communityCatalogType {
```

Comment:
I think we still need this. We don't want to create entries for the Helm stable catalog as its deprecated.

Reply:
Even though we don't have any generated `AppCatalogEntry` belongs to helm-stable, yes, you're right. I will revert this back.